packages/vitest/src/integrations/snapshot/chai.ts

```diff
@@ -69,7 +67,7 @@ export const SnapshotPlugin: ChaiPlugin = (chai, utils) => {
           throw new Error(`${key} cannot be used with "not"`)
         }
         const expected = utils.flag(this, 'object')
-        const test = utils.flag(this, 'vitest-test')
+        const test: Test = utils.flag(this, 'vitest-test')
```

Comment:
Do we need to assert it? Or do we always have it?

Reply:
This can be undefined when concurrent tests messes up `getCurrentTest()` or simply outside of `test`. I don't think it would've worked before, so we should throw an explicit error like `expect.soft` https://github.com/vitest-dev/vitest/blob/99c64e64122b4ad49bd2588330df4da0be59e032/packages/expect/src/utils.ts#L41-L45